packages/mysql/src/Adaptor.js

```diff
@@ -12,6 +12,40 @@ import squel from 'squel';
  * @param {Operations} operations - Operations to be performed.
  * @returns {Operation}
  */
+export function sql(sqlQuery, options, callback) {
```

Comment:
Please remove `callback` param, We are dropping out callback support. Instead users should be to use `.then()` and ` .catch()` when using this function